misc/labels.md

```diff
@@ -0,0 +1,99 @@
+# How we use github labels
+
+*Because only team-mermbers can add and change labels this document is mainly for maintainers, but also for users to understand how we use labels.*
+
+*It is important to also label old and closed issues uniformly to be able to export them later r.g. if the project gets separated into multiple components.*
+
+
+## Issue Types
+If an issue was created it always MUST be labeled as one of the following issue types:
+
+### `Question`
+The author has a general or very specific question.<br>
+If it is a general question how to use vis.js the issues should be closed immediately with a reference to [stackoverflow](https://stackoverflow.com/questions/tagged/vis.js).<br>
+Specific question or hard problems should stay open.<br>
+Questions should be closes within 3 month.
+
+### `Problem`
+This issues points to a potential bug that needs to be confirmed.<br>
+If the problem is probably in the users code it should be better labeled as `Question`.<br>
+The support-Team should try to reproduce this issue and than close it or mark it as `Confirmed Bug`:<br>
+`Problem` -> `Confirmed Bug` --> `Work In Progress` --> `Fixed awaiting release`
+
+### `Feature-Request`
+This issue proposes a new feature or a change of existing functionality. Issues that are very very unlikely to get implemented should be closed.<br>
+This label is often used together with `Requires breaking change`: <br>
+`Feature-Request` --> `Work In Progress` --> `Fixed awaiting release`
+
+### `wontfix`
+This issues is e.g. only for discussing a topic and is not handled in the usual issue process.
+
+
+## Graph type
+All issues MUST have one of the following type-labels. This labels usually are usually mutually exclusive:
+
+### `Network`
+Concerns the Timeline-Graph implementation.
+
+### `Timeline`
+Concerns the Timeline-Graph implementation.
+
+### `Graph2D`
+Concerns the 2D-Graph implementation.
+
+### `Graph3D`
+Concerns the 3D-Graph implementation.
+
+### `DataSet`
+Concerns the DataSet implementation.
+
+### `Docs`
+This issue concerns only the documentation.<br>
+If an existing issue is documented wrongly this is a `Problem` in the component and not a `docs` issue.<br>
+This can be used for typos or requests for a improvement of the docs.
+
+### `source/non-public API`
+This issues is just for discussion or is concerning the build-process, the code-style or something similar.
+
+
+## Additional labels
+
+### `Duplicate`
+This issues is a duplicate of an existing issue. It should be closed naming the original issue it is a duplicate of.
```

Comment:
Improvement:

It should be closed naming the original issue it is a duplicate of.

-->  The duplicate should be closed. In addition, add a reference to the original issue with a comment.